aea/configurations/base.py

```diff
@@ -643,6 +881,23 @@ def __init__(
             self.logging_config["version"] = 1
             self.logging_config["disable_existing_loggers"] = False
 
+    @property
+    def package_dependencies(self) -> Set[ComponentId]:
+        """Get the package dependencies."""
+        protocols = set(
+            ComponentId(ComponentType.PROTOCOL, public_id)
+            for public_id in self.protocols
+        )
+        connections = set(
+            ComponentId(ComponentType.CONNECTION, public_id)
+            for public_id in self.connections
+        )
+        skills = set(
+            ComponentId(ComponentType.CONNECTION, public_id)
+            for public_id in self.skills
+        )
```

Comment:
Contracts?

Reply:
Even though this is against develop, you mention contracts above, so maybe just add a todo